package.json

```diff
@@ -83,7 +84,7 @@
     "tsc-watch": "^1.0.10",
     "tslint": "^5.0.0",
     "typescript": "^2.5.3",
-    "web3": "^1.0.0-beta.24",
+    "web3": "^1.0.0-beta.26",
```

Comment:
If you update dependencies, make it as part of the new PR! this could break things

Reply:
Will consider it next time.

Reply:
@michaelScoff yes and no. when it's low priorty updates like from 1.0.0 to 1.0.1 it's ok. as long as it doesn't upgrade major updates like 1.1 to 1.2.0